app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,22 @@
+Spree::Order.class_eval do
+  checkout_flow do
+    go_to_state :address, :if => lambda { |order| order.address_required? } 
```

Comment:
Are `go_to_state :address` and `go_to_state :payment` really needed here if their lambdas always return false?

Reply:
You are right, those are not needed.

Reply:
Already Removed